src/Classes/Particle/Linker.ts

```diff
@@ -62,4 +62,4 @@ export class Linker {
             }
         }
     }
-}
+}
```

Comment:
keep the newline at the end of file, Codacy will drop quality for it

Reply:
No worries, my linter must've changed it